delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/Projects.scala

```diff
@@ -159,6 +165,44 @@ trait Projects {
     * @param offset the last seen event offset; it will not be emitted by the stream
     */
   def currentEvents(offset: Offset = NoOffset): Stream[Task, Envelope[ProjectEvent]]
+
+  /**
+    * Create a project for a user according to automatic provisioning settings.
+    *
+    * @param subject  the user for which to provision the project.
+    */
+  def provisionProject(subject: Subject): UIO[Unit]
+
+  private[delta] def provisionIfNotExists(
```

Comment:
The return signature is weird: `IO[Unit, Unit]`...it does not give any information on the error . Plus, the only place where you are using provisionIfNotExists is doing `provisionIfNotExists.onErrorHandle(_ => ())`. So you aren't using the error channel at all in this case.